go/vt/vttablet/tabletmanager/vreplication/controller.go

```diff
@@ -41,16 +40,6 @@ import (
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 )
 
-var (
```

Comment:
@deepthi, is it safe to remove the healthcheck flags? They were deprecated several versions earlier but do we want to play safe and leave them in for now?

Reply:
Actually there are several tests that depend on them, so added them back